src/utils.ts

```diff
@@ -19,3 +20,55 @@ export const getConfig = db
 	.from(Config)
 	.where(eq(Config.id, sql.placeholder("guildId")))
 	.prepare();
+
+// TEMP: use .all() and select the first row manually, .get() does not work
```

Comment:
Is this still necessary? I would double-check.

Reply:
For me it is, I ran into the same issue earlier where the returned value was `undefined`. It might be an issue with bun or SQLite since I don't have this behaviour in other projects with drizzle.